system-test/cluster.ts

```diff
@@ -0,0 +1,313 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {afterEach, beforeEach, describe, it} from 'mocha';
+import {generateId} from './common';
+import {
+  Bigtable,
+  ClusterInfo,
+  Instance,
+  Cluster,
+  GetClusterMetadataResponse,
+} from '../src';
+import assert = require('assert');
+import {ClusterUtils} from '../src/utils/cluster';
+
+describe('Cluster', () => {
+  const bigtable = new Bigtable();
+  let instance: Instance;
+
+  async function createNewInstance(clusters: ClusterInfo[]): Promise<void> {
+    const instanceId: string = generateId('instance');
+    instance = bigtable.instance(instanceId);
+    const [, operation] = await instance.create({
+      clusters,
+      labels: {
+        time_created: Date.now(),
+      },
+    });
+    await operation.promise();
+  }
+  async function createStandardNewInstance(
+    clusterId: string,
+    nodes: number
+  ): Promise<void> {
+    return await createNewInstance(standardCreationClusters(clusterId, nodes));
+  }
+  function standardCreationClusters(
+    clusterId: string,
+    nodes: number
+  ): ClusterInfo[] {
+    return [
+      {
+        id: clusterId,
+        location: 'us-east1-c',
+        nodes,
+      },
+    ];
+  }
+  afterEach(async () => {
+    await instance.delete();
+  });
+  describe('Create cluster', () => {
+    describe('With manual scaling', () => {
+      let clusterId: string;
+      let cluster: Cluster;
+      async function checkMetadata(localCluster: Cluster, nodes: number) {
+        const metadata: GetClusterMetadataResponse =
+          await localCluster.getMetadata({});
+        assert.strictEqual(metadata[0].serveNodes, nodes);
+      }
+      beforeEach(async () => {
+        clusterId = generateId('cluster');
+        await createStandardNewInstance(clusterId, 2);
+        cluster = instance.cluster(clusterId);
+      });
+      it('Create an instance with clusters for manual scaling', async () => {
+        await checkMetadata(cluster, 2);
+      });
+      it('Create an instance and then create a cluster for manual scaling', async () => {
```

Comment:
nit - here and in all other `it(..` descriptions, I think they should be reworded slightly; for example, here I would change it to say:

`it('should create an instance and then create a cluster with manual scaling')`

Let me know your thoughts :)

Reply:
Very good point. I'll look at some of the other PRs and make sure that they have these descriptions as well.